src/rez/cli/test.py

```diff
@@ -8,15 +8,26 @@ def setup_parser(parser, completions=False):
     parser.add_argument(
         "-l", "--list", action="store_true",
         help="list package's tests and exit")
+    parser.add_argument(
+        "--dry-run", action="store_true",
+        help="dry-run mode: show what tests would have been run, but do not "
+        "run them")
+    parser.add_argument(
+        "-s", "--stop-on-fail", action="store_true",
+        help="stop on first test failure")
+    parser.add_argument(
+        "--inplace", action="store_true",
```

Comment:
Should `--extra-packages` and `--inplace` be mutually exclusive? In theory it doesn't make much sense to let a user use `--extra-package` if it's gonna be ignored... Or would it be ignored right now?

Reply:
Not sure, but agreed, they should be mutually exclusive to avoid confusion.